server/views/layout.pug

```diff
@@ -71,6 +71,17 @@ html(lang="en")
 
     main.app-container(role="main")
       include ./includes/notice
+
+      div(class="change-location")
+        if user.activeCaseLoad && !user.activeCaseLoad.description.includes('---')
+          span(class="change-location__location" data-qa="active-location") #{user.activeCaseLoad.description}
+
+        if user.activeCaseLoads && user.activeCaseLoads.length > 1
```

Comment:
rather than define the same checks multiple times it would probably read better to extract out into readable variable names

`if hasMultipleCaseloads`
`if hasActiveCaseload`

`if hasMultipleCaseloads || hasActiveCaseload`

there should be other examples of how to do this in licences

Reply:
goos idea